lib/ddtrace/metrics.rb

```diff
@@ -247,16 +256,18 @@ def gauge(stat, value, options = nil)
     INCOMPATIBLE_STATSD_ONLY_ONCE = Datadog::Utils::OnlyOnce.new
     private_constant :INCOMPATIBLE_STATSD_ONLY_ONCE
 
-    # IMPORTANT: Once you delete this, you probably want to update the `#ignored_statsd_warning` below where we
-    # recommend that customers add `gem 'dogstatsd-ruby', '~> 4.0'` to their Gemfile to perhaps state something
-    # different.
```

Comment:
⬆️  You forgot to update the `#ignored_statsd_warning` method that still recommends users install ~> 4.0 🤣

Reply:
Damn, thanks!